sci-physics/geant-vmc/geant-vmc-9999.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vmc-project/geant4_vmc.git"
+else
+	DOWN_PV=$(get_version_component_range 2-)
+	SRC_URI="http://root.cern.ch/download/vmc/geant4_vmc.${DOWN_PV}.tar.gz"
+	SOURCE_PV=$(replace_all_version_separators - ${DOWN_PV})
+	S="${WORKDIR}/geant4_vmc-${SOURCE_PV}"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Virtual Monte Carlo Geant4 implementation"
+HOMEPAGE="http://root.cern.ch/root/vmc/VirtualMC.html"
+
+LICENSE="GPL-2"
+SLOT="4"
```

Comment:
This is a 9999 ebuild. Please remove the `SLOT`.

Reply:
The intent of the SLOT operator was to differentiate this live ebuild from the Geant3-VMC code, which pairs with sci-physics/geant:3 (not packaged anymore...). 

If geant:3 was still alive, also `=geant-vmc-9999:3` might be a packaged version which could coexist with `=geant-vmc-9999:4` (both would build from a separate repository and depend on matching `sci-physics/geant` SLOTs). 

So it is more like e.g. `=sys-libs/compiler-rt-sanitizers-9999:7.0.0` which currently lives in tree. 

Should the SLOT still be removed? 
Note that
```
EGIT_REPO_URI="https://github.com/vmc-project/geant4_vmc.git"
```
and a future `GeantV` version of VMC (no idea if this will be made) would for sure be in a separate repository again, and could coexist for a while.